lock/redis/standalone_redis_lock.go

```diff
@@ -0,0 +1,196 @@
+/*
```

Comment:
`lock/redis/standalone.go` is ok to  me.

Reply:
fixed